openapi/components/schemas/AML.yaml

```
@@ -0,0 +1,165 @@
+type: object
+properties:
+  firstName:
+    type: string
+    description: First Name
+    example: Benjamin
+  lastName:
+    type: string
+    description: Last Name
+    example: Franklin
+  source:
+    type: string
+    readOnly: true
+    description: Which list this came from.
+  sourceType:
+    readOnly: true
+    description: The list type
+    type: string
+    enum:
+      - pep
+      - sanctions
+      - adverse-media
+  gender:
+    type: string
+    readOnly: true
+    nullable: true
+    description: Gender of returned identity (if `type` is `individual`)
+  title:
+    type: array
+    readOnly: true
+    nullable: true
+    description: The title of their position.
+    example: President
```

This is supposed to be an array of strings.
```suggestion
    example: 
      - Postmaster General
      - Ambassador to France
      - Ambassador to Sweden
      - Supreme Executive Council of Commonwealth of Pennsylvania
```